smoke/react/src/routes/Todo.tsx

```
@@ -32,14 +32,14 @@ export default function TodoList() {
         />
         <button
           onClick={() => {
-            console.log(`saving todo-0: ${todo.text}`);
+            // console.log(`saving todo-0: ${todo.text}`);
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             delete (todo as any)._id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)